src/communication/xcWebSocketPublisher.ts

```diff
@@ -52,17 +52,16 @@ export class DefaultPublisher implements Publisher {
         };
     }
 
-    private getHeaderConfig(componentCode: number, stateMachineCode: number, messageType: string, visibilityPrivate: boolean, specifiedPrivateTopic: string, stateMachineId: number = undefined, agentId: number = undefined): Header {
+    private getHeaderConfig(componentCode: number, stateMachineCode: number, messageType: string, visibilityPrivate: boolean, specifiedPrivateTopic: string, stateMachineId: number = undefined): Header {
```

Comment:
do not change interfaces for now. we'll need to support both versions of xcomp for now